qemu/tests/rh_qemu_update.py

```diff
@@ -0,0 +1,232 @@
+                    host_qemu["nvr"],
+                    guest_qemu["nvr"]
+                )
+            )
+        return host_qemu['nvr']
+
+    def update_guest_qemu(session, install_id):
+        """
+        Prepare repository containing the newest version of qemu-kvm package,
+        handle modules if needed and run upgrade to the newest version
+        """
+        def nice_format(ugly_format):
+            better_format = ugly_format.replace("  ", " ")
+            if "  " in better_format:
+                return nice_format(better_format)
+            else:
+                return better_format
+        if "+" in install_id:
+            build_or_module = "cm_cmd_module"
+        else:
+            build_or_module = "cm_cmd_brew"
+        # Prepare module or build repo containing newer version of qemu-kvm
+        cmd = f"python3 {params['cm_path']}{params[build_or_module]} {install_id}"
+        test.log.info(f"Running: {nice_format(cmd)}")
+        try:
+            session.cmd(cmd, timeout=OPERATION_TIMEOUT)
+            test.log.info("Creation of repo '{}' succeeded".format(
+                install_id
+                )
+            )
+        except Exception as e:
+            test.error("Creation of repo '{}' failed with: {}".format(
+                install_id, e
+                )
+            )
+        # Disable and enable new module if module is used
+        if "+" in install_id:
+            # Get virt module stream ('rhel' or 'av') on the host
+            stream = process.run(
+                f"{params['pm_tool']} module list --enabled | grep virt" +
+                "| awk -F ' ' '{{print $2}}' | head -1",
+                shell=True
+            ).stdout_text.strip()
+            disable_cmd = f"{params['pm_tool']} module --disable virt"
+            s_disable, o_disable = session.cmd_status_output(
+                disable_cmd, timeout=OPERATION_TIMEOUT
+                )
+            if s_disable != 0:
+                test.fail("Disable of module virt failed with: {}".format(
+                    o_disable
+                    )
+                )
+            else:
+                test.log.info("Disable of module virt succeeded")
+            enable_cmd = f"{params['pm_tool']} module --enable virt:{stream}"
+            s_enable, o_enable = session.cmd_status_output(
+                enable_cmd, timeout=OPERATION_TIMEOUT
+                )
+            if s_enable != 0:
+                test.fail("Enable of module virt:{} failed with: {}".format(
+                    stream, o_enable
+                    )
+                )
+            else:
+                test.log.info("Enable of module virt:{} succeeded".format(
+                    stream
+                    )
+                )
+        # Run upgrade to newer qemu-kvm version
+        if "+" in install_id:
+            cmd_upgrade = f"{params['pm_tool']} module update -y virt"
```

Comment:
Miss `stream`

Reply:
If I am not wrong it should be fine if proper module is already enabled, but I agree let's keep it specific.

Reply:
Done.